book/modules/appendix1-exercises.tex

```diff
@@ -1,7 +1,57 @@
-
 \begin{exercises}
 
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is linear. If so, write it in its standard form. If not, explain what makes it nonlinear.
+    \begin{enumerate}
+      \item $\cos(4)x_1+\mathrm{e}y_2+\pi z_3=\mathrm{e}^{\pi}$
+      \item $4x_1+2x_2+5x_4=4x_2+4x_5+5$
+      \item $12x+3xy+5z=2$
+      \item $\sin(4)x+\cos(4)y+\tan(4)z=\operatorname{sec}(4)x+\operatorname{cosec}(4)y+\operatorname{cot}(4)z$
+      \item $5x+2y+8z=\cos(y)$
+    \end{enumerate}
+    
+    \prob For each system of linear equations given below, (a) write down its augmented matrix; (b) use row reduction algorithm to determine if it is consistent or not; (c) for each consistent system, give the complete solution set.
```

Comment:
`complete solution`, or `solution set`, but not both. Change this to a `: (a) ...., (b) ...`. That is, a colon then a list separated by commas.